workflow-bot-app/src/test/java/com/symphony/bdk/workflow/SendMessageIntegrationTest.java

```diff
@@ -33,6 +33,7 @@
 import java.util.Collections;
 import java.util.List;
 
+@SuppressWarnings("unchecked")
```

Comment:
We can also put the annotation directly on the var declaration to avoid it being applied everywhere